src/languages/es.js

```diff
@@ -311,11 +311,9 @@ export default {
         new: {
             welcome: 'Bienvenido/a',
             chooseAName: 'Elige un nombre',
-            helpText: '¿Necesitas ayuda con la configuración? Pídenos una llamada y una persona de nuestro equipo te ayudará.',
+            helpText: '¡Dale un nombre a tu Workspace antes de activar tus Expensify Cards!',
             getStarted: '¡Empezar!',
-            uploadPhoto: 'Subir Foto',
-            editPhoto: 'Editar Foto',
-            requestCall: 'Concertar una llamada',
+            genericFailureMessage: 'Se ha producido un error al intentar crear el Workspace. Por favor intentalo de nuevo.',
```

Comment:
@HorusGoul do you speak Spanish or is this Google Translate? If it's the latter we should have a Spanish-speaking Expensifier verify that the grammar is good.

Reply:
I'm a native speaker! Also, I just sent a commit fixing a typo.